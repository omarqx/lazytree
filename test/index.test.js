const treeview = require('../src/index');

describe('basic list', () => {
  it('should render basic two elements correctly', () => {
    const tree = treeview([
      {
        value: 'first',
        id: 1
      },
      {
        value: 'second',
        id: 2
      }
    ]);
    expect(tree).toMatchSnapshot();
  });
  it('should render basic single element correctly', () => {
    const tree = treeview([
      {
        value: 'first',
        id: 1
      }
    ]);
    expect(tree).toMatchSnapshot();
  });
});