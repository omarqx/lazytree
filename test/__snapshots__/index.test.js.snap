// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic list should render basic single element correctly 1`] = `
<ul>
  <li
    id="1"
  >
    first
  </li>
</ul>
`;

exports[`basic list should render basic two elements correctly 1`] = `
<ul>
  <li
    id="1"
  >
    first
  </li>
  <li
    id="2"
  >
    second
  </li>
</ul>
`;
